entrypoint.sh.addHostUserToContainer

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# Add local user
+# Either use the LOCAL_USER_ID if passed in at runtime or
+# fallback
+
+USER_ID=${LOCAL_USER_ID:-9001}
+
+echo "Starting with UID : $USER_ID"
+adduser -s /bin/sh -h /home/user -u $USER_ID -D user
+export HOME=/home/user
+addgroup user holochain
+addgroup user sudoers
+su -c  "chown user:holochain /home/user -R"
+
+exec su-exec user "$@"
```

Comment:
Now I get it. This magic will run what you supply on the run command line.

I guess with no args this will still give you a shell, otherwise the container stops when this script ends.